Configuration/AlCa/python/autoCondModifiers.py

```diff
@@ -115,7 +115,7 @@ def autoCondRelValForRun3(autoCond):
 
     GlobalTagRelValForRun3 = {}
     L1GtTriggerMenuForRelValForRun3 =    ','.join( ['L1Menu_Collisions2015_25nsStage1_v5' , "L1GtTriggerMenuRcd",             connectionString, "", "2023-01-28 12:00:00.000"] )
-    L1TUtmTriggerMenuForRelValForRun3 =  ','.join( ['L1Menu_Collisions2025_v1_2_0_xml'    , "L1TUtmTriggerMenuRcd",           connectionString, "", "2025-06-13 12:00:00.000"] )
+    L1TUtmTriggerMenuForRelValForRun3 =  ','.join( ['L1Menu_Collisions2025_v1_3_0_xml'    , "L1TUtmTriggerMenuRcd",           connectionString, "", "2025-06-13 12:00:00.000"] )
```

Comment:
```suggestion
    L1TUtmTriggerMenuForRelValForRun3 =  ','.join( ['L1Menu_Collisions2025_v1_3_0_xml'    , "L1TUtmTriggerMenuRcd",           connectionString, "", "2025-07-24 05:13:35.000"] )
```

@perrotta, the snapshot here is earlier than the tag creation (see also https://github.com/cms-sw/cmssw/commit/430f1ff382ba10977f81f2f62010d219177f0687).